jobbergate-cli/tests/test_config.py

```diff
@@ -17,6 +17,7 @@ def test_SENTRY_TRACE_RATE__requires_float_in_valid_range():
         Settings(SENTRY_TRACE_RATE=1.1)
 
 
+@pytest.mark.skip(reason="There are no required fields in the Settings class at the moment")
```

Comment:
Let's just remove this test since there are now no required fields.

Reply:
Test removed by 894e6077f4ff2a0fe39a38ad50f6d87421af0bab.